server/src/main/java/com/tencent/rss/server/LocalStorageChecker.java

```diff
@@ -61,11 +61,14 @@ public LocalStorageChecker(ShuffleServerConf conf) {
   @Override
   public boolean checkIsHealthy() {
     int num = 0;
+    long totalLocalDataSize = 0;
     for (StorageInfo storageInfo : storageInfos) {
       if (storageInfo.checkIsHealthy()) {
         num++;
       }
+      totalLocalDataSize += getUsedSpace(storageInfo.storageDir);
```

Comment:
getUsedSpace is called in storageInfo.checkIsHealthy(), reuse it if it's a heavy call which impact performance

Reply:
OK, add a filed named usedSize in the StorageInfo？